setup-miarec.yml

```diff
@@ -12,10 +12,10 @@
   pre_tasks:
     - include_vars: vars/custom.yml
       failed_when: false
-      
+
     - set_fact:
-        miarecweb_db_host: "{{ hostvars[groups.db.0].private_ip_address }}"
-        miarecweb_redis_host: "{{ hostvars[groups.redis.0].private_ip_address }}"
+        miarecweb_db_host: "{{ db_host_service if db_host_service is defined else hostvars[groups.db.0].private_ip_address }}"
+        miarecweb_redis_host: "{{ redis_host_service if redis_host_service is defined else hostvars[groups.redis.0].private_ip_address }}"
```

Comment:
Where are the variables `db_host_service`/`redis_host_service` defined and used?

Reply:
This was built around the same time as [PR #2 ](https://github.com/miarec/ansible-miarec/pull/2)  which would allow for and external db or redis service like Amazon RDS, and Elasticache    its not required for this PR , this should be removed here